src/sql/workbench/contrib/profiler/browser/profilerEditor.ts

```diff
@@ -609,6 +613,9 @@ export class ProfilerEditor extends EditorPane {
 	}
 
 	private _updateToolbar(): void {
+		console.log("this.input.state.isRunning:", this.input.state.isRunning);
```

Comment:
Please remove these - if you want to keep log statements in they should use the log service instead